.vscode/launch.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
Yeah, let's ignore the `.vscode` directory since people will have different setups and we usually recommend the jest-runner extension for those using vscode

Reply:
Sweet, I'll scrub it